flow/environment.py

```diff
@@ -59,7 +60,16 @@ def setup(py_modules, **attrs):
     an environment's module, but also register it with the global signac
     configuration. Once registered, the environment is automatically
     imported when the :meth:`~flow.get_environment` function is called.
+
+    Warning
+    -------
```

Comment:
numpydoc [prefers the title "Warnings" here](https://numpydoc.readthedocs.io/en/latest/format.html#warnings) but suggests other [methods for deprecation notices](https://numpydoc.readthedocs.io/en/latest/format.html#deprecation-warning).

However, I don't think this notice is needed at all. You could leave it to just the `FutureWarning`.